Makefile

```diff
@@ -30,6 +30,8 @@ deps: clean
 	yarn
 	# Call the build script for the adapter if defined (TypeScript adapters have this extra build/compile step)
 	# We use `wsrun` to build workspace dependencies in topological order (TODO: use native `yarn workspaces foreach -pt run build` with Yarn 2)
+	if [ $(adapter) = bootstrap ]; then yarn wsrun -mre -p @chainlink/ea-bootstrap -t build; fi
```

Comment:
@krebernisak Can you think of a cleaner way to do this?

Creating new make commands & github strategies for `adapters-core` seemed too verbose, but maybe that is a better path.